src/createProxy.ts

```diff
@@ -134,11 +134,15 @@ export async function createProxy(
   addRoutes(server, proxyConfig);
   server.events.on('log', (event, tags) => {
     if (tags.error) {
-      log.error(
-        `Server error: ${
-          event.error ? (event.error as any).message : 'unknown'
-        }`
-      );
+      let message = event.error ? (event.error as any).message : 'unknown';
+      if (message.includes('alert bad certificate')) {
+        message = `${message}
+        It appears that the certificate has not been trusted yet. intervene tries to automatically trust the HTTPS certificate
+        but it does not always work for all browsers (e.g. Firefox).
+        To trust the certificate, open the requested URL in your browser and add a security exception for the certificate.
```

Comment:
This is great, but I think we need to change the message a bit.

What has happened is that the server has been accessed by something that does not trust the certificate. That could be `curl`, Firefox, Chrome on Linux if the certutils aren't installed, or something else entirely.

Could we make this something like:
```
intervene attempts to trust the certificate with the operating system when supported. This works for some browsers but not all (e.g. Firefox). For Firefox, open the requested URL in your browser and add a security exception for the certificate. For Chrome on Linux, make sure you have `certutil` available - package `libnss3-tools` under Debian based distributions (e.g. Ubuntu) and restart intervene.  For other browsers or operating systems, refer to your browser or tool documentation about accessing URLs with untrusted certificates.
```